x/distribution/client/cli/tx.go

```diff
@@ -68,16 +94,17 @@ $ <appcli> tx distr withdraw-rewards cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs
 				msgs = append(msgs, types.NewMsgWithdrawValidatorCommission(valAddr))
 			}
 
-			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, msgs)
+			return splitGenerateOrBroadcast(cliCtx, txBldr, msgs)
```

Comment:
What about the REST endpoint?

Reply:
Resolving as we already discussed in Slack